File: Python/reverse.py
strs = input("Enter the String: ")
str2 = ""
for i in range(len(strs)-1, -1, -1):
    str2+=strs[i]

if strs == str2:
    print("String is Palindrome.")
else:
    print("String is not Palindrome.")
# print(str2)

File: Java/reverse.java
package Java;
import java.util.Scanner;

public class reverse {
    public static void main(String args[])
    {
        System.out.print("Enter the String: ");
        Scanner sc = new Scanner(System.in);
        String str = sc.nextLine();
        // System.out.println(str);
        String str2 = "";
        for(int i=str.length()-1; i>=0; i--)
        {
            str2 = str2 + str.charAt(i);
        }
        System.out.println(str2);

    }
}
